horovod/spark/keras/datamodule.py

```diff
@@ -0,0 +1,173 @@
+# Copyright (C) 2022, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+import contextlib
+import horovod.tensorflow.keras as hvd
+
+from horovod.spark.common import constants
+from horovod.spark.common.datamodule import DataModule
+
+
+class PetastormDataModule(DataModule):
+    """Default Petastorm-based DataModule for KerasEstimator."""
+    def __init__(self, reader_pool_type: str="process",
```

Comment:
reader_pool_type: str='thread'.

To be consistent with default value here: https://github.com/horovod/horovod/blob/master/horovod/spark/common/params.py#L167